CMakeLists.txt

```diff
@@ -263,6 +263,11 @@ if (OBJCOPY_PATH AND YANDEX_OFFICIAL_BUILD AND (NOT CMAKE_TOOLCHAIN_FILE))
     set (USE_BINARY_HASH 1)
 endif ()
 
+# Allows to build stripped binary in a separate directory
+if (OBJCOPY_PATH AND READELF_PATH)
```

Comment:
Does it mean that `BUILD_STRIPPED_BINARIES_PREFIX` will be always set according to https://github.com/ClickHouse/ClickHouse/pull/35196/files#diff-20dc468ff5043ade1489346dbd70f621d828a1c8c6a439ef9b0afc3bc295a0a0R178 ?

I don't really think we always want strip it

Reply:
Figured it out in slack